README.md

```diff
@@ -112,6 +113,45 @@ export default ({store}) => {
 }
 ```
 
+#### Using cookies (universal client + server-side)
+Add `cookie` and `js-cookie`:
+
+`npm install --save cookie js-cookie`
+or `yarn add cookie js-cookie`
+
+```javascript
+...
+plugins: [{ src: '~/plugins/persistedState.js']
+...
+```
+
+```javascript
+// ~/plugins/persistedState.js
+
+import createPersistedState from 'vuex-persistedstate';
+import * as Cookies from 'js-cookie';
+import cookie from 'cookie';
+
+export default ({ store, req }) => {
+    createPersistedState({
+        paths: [...],
+        storage: {
+            getItem: (key) => {
+                if (process.server) {
```

Comment:
Can you add a comment + link about this process flag? (https://nuxtjs.org/guide/plugins/#using-process-flags)